src/Simplex/Chat/Types.hs

```diff
@@ -67,8 +72,11 @@ type UserId = ContactId
 
 type ContactId = Int64
 
+type ProfileId = Int64
+
 data Contact = Contact
   { contactId :: ContactId,
+    contactProfileId :: ProfileId,
```

Comment:
this all seems like a hack, to be honest. A better solution would be to create another type for profile used internally - e.g. LocalProfile, or DbProfile, and include profile ID there.

As is, it's not clear what this ID is related to